src/Main.hs

```diff
@@ -58,27 +91,100 @@ run (TransformFlags i o c) = do
         (_     , Left e) -> Left e
         (Left e, _     ) -> Left e
 
-  putStrLn $ "RESOLVED:\n" ++ ppShow resolvedASTTable
+  when verbose $ do
+    putStrLn $ "OUTPUT: " ++ outputPath
+    putStrLn $ "ENTRYPOINT: " ++ canonicalEntrypoint
+    putStrLn $ "ROOT PATH: " ++ rootPath
+  when debug $ do
+    putStrLn $ "PARSED:\n" ++ ppShow astTable
+    putStrLn $ "RESOLVED:\n" ++ ppShow resolvedASTTable
```

Comment:
Totally good for now. In the future, we might want to use `katip` or something similar: https://github.com/Soostone/katip/tree/master/katip